db/migrate/20240215210244_create_template_versions.rb

```diff
@@ -7,7 +7,10 @@ def change
       t.json :version_diff, default: {}
       t.date :version_date, null: false
       t.integer :status, default: 0
-      t.references :requirement_template, foreign_key: true, type: :uuid, index: true
+      t.references :requirement_template,
+                   foreign_key: true,
+                   type: :uuid,
+                   index: true
 
       t.timestamps
```

Comment:
@taw27 - can we add something here or on the seeding that seeds a published version after the requirements are loaded so it's publshed from the previous day?  This would make sure there is one active Low Density - New construction checklist

Reply:
ok will take a look